docs/docs/constraints.md

```diff
@@ -29,6 +29,10 @@ The constraint lets you specify:
 5. The security level of the instance: `SECURE`, `STALE`, `INSECURE`
 6. The maximum age of the attestation in ISO-8601 duration format
 
+!!! note
+    Clients that set the security level to `INSECURE` can only connect to `INSECURE` enclaves. This prevents clients from
+    accessing enclaves running in production by mistake.
```

Comment:
Please rewrite as:

"This prevents clients from accessing production enclaves by mistake."